rest-api/tools/deploy_app.sh

```diff
@@ -177,4 +177,14 @@ then
   rm "${tmp_files[@]}"
 fi
 
+test_request=$(curl -s http://localhost:8080/rdr/v1/ | grep version_id)
```

Comment:
Sorry, you can't use localhost here. You have to use the host of the server.

Reply:
(Please test this by deploying to sandbox.)

Reply:
In particular, replace http://localhost:8080 with https://${PROJECT}.appspot.com. (Note that we always use SSL.)